src/main/java/greencity/entities/OpeningHours.java

```diff
@@ -0,0 +1,37 @@
+package greencity.entities;
+
+import greencity.entities.enums.WeekDays;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.time.LocalTime;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OpeningHours {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @DateTimeFormat(pattern = "HH:mm")
+    private LocalTime openTime;
+
+    @Column(nullable = false)
```

Comment:
Be careful with this formatting. 
It can throw ConversionFailedException during serialisation or deserialisation. Or both.
If this is just for nice date presentation - you can change it at front-end. 
If this have wider purpose - check this very well. Or think about some Converter